core/domain/user_id_migration_test.py

```diff
@@ -1027,6 +1027,9 @@ def test_one_user_one_model_full_id(self):
             entity_type='type',
             entity_id='id',
             subject='subject').put()
+        feedback_models.GeneralFeedbackThreadUserModel(
+            id='None.thread_id',
```

Comment:
I can't remember -- is there an issue filed somewhere to stop this from happening?

Reply:
#8202